xml/chapter1/section2/subsection6.xml

```diff
@@ -798,6 +847,34 @@ timed_prime_test(43);
     test primes near 1000?  Do your data bear this out?  Can you explain any
     discrepancy you find?
     <LABEL NAME="ex:mod-timed-prime-test"/>
+        <SOLUTION>
+      <SNIPPET>
+	<NAME>mod_timed_prime_test_solution</NAME>
+	<REQUIRES>fast_prime_definition</REQUIRES>
+	<JAVASCRIPT>
+  function timed_prime_test(n) {
+      display(n);
+      return start_prime_test(n, runtime());
+  }
+  function start_prime_test(n, start_time) {
+      return fast_is_prime(n, math_floor(math_log(n)))
+            ? report_prime(runtime() - start_time)
+            : true;
+  }
+  function report_prime(elapsed_time) {
+      display(" *** ");
+      display(elapsed_time);
+  }
+        </JAVASCRIPT>
+        <SCHEME>
+        </SCHEME>
+        The time to test primes near 1,000,000 using <JAVASCRIPTINLINE>fast_is_prime</JAVASCRIPTINLINE> 
```

Comment:
This text is currently ignored, because it is inside the snippet tag. Move it just behind the ending </SNIPPET>.